api/client.go

```diff
@@ -24,6 +25,65 @@ type ClientAPI struct {
 	NotifyDeleted func(uint, string)
 }
 
+// UpdateClient updates a client name by its id.
+// swagger:operation PUT /client/{id} client updateClient
+//
+// Update a client.
+//
+// ---
+// consumes: [application/json]
+// produces: [application/json]
+// security: [clientTokenHeader: [], clientTokenQuery: [], basicAuth: []]
+// parameters:
+// - name: body
+//   in: body
+//   description: the client name to update
```

Comment:
While it's true that only the name can be updated, this variable refers to an object which contains the name.
```suggestion
//   description: the client to update
```